auth-web/src/components/auth/manage-business/CreateNewBusinessHelp.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <section class="help-content">
+    <header class="d-flex justify-center mt-4">
+      <p class="title-text mt-4">{{ helpContent.title }}</p>
+    </header>
+    <div class="help-text mt-4">
+      <p>{{ helpContent.intro }}</p>
+      <ol>
+        <li class="mt-12" v-for="(step, index) in helpContent.steps" :key="index">
+          <strong>{{ step.title }}</strong>
+          <ul class="mt-8">
+            <li v-for="(item, i) in step.items" :key="i">
+              <p v-if="typeof item === 'string'">{{ item }}</p>
+              <p v-else v-html="item.content"></p>
+            </li>
+          </ul>
+        </li>
+      </ol>
+    </div>
+  </section>
+</template>
+
+<script>
+export default {
+  data () {
+    return {
+      helpContent: {
+        'title': 'Help with Starting a Business',
+        'intro': 'Start a named or numbered business in B.C. by following these steps:',
+        'steps': [
+          {
+            'title': 'Decide on a Business Type',
+            'items': [
+              {
+                'content': "Decide which business structure is the most appropriate for you. A few options are: a sole proprietorship, partnership, or corporation. Each structure has different legal and financial implications. <a href='https://entity-selection-prod.apps.silver.devops.gov.bc.ca/'>Use the Business Structures Wizard to help you decide.</a>",
+                'isHtml': true
+              },
+              'If you want to start a corporation, you also have the choice of using a named or numbered company.'
+            ]
+          },
+          {
+            'title': 'Request a Business Name or Use a Numbered Company',
+            'items': [
+              'Both named and numbered companies are started by selecting the "Get Started with a B.C.-based business button.',
+              'If you would like to start a named business you must first submit a name request. Create a unique name for your business and submit this name for examination by the Business Registry.'
+            ]
+          },
+          {
+            'title': 'Incorporate or Register',
+            'items': [
+              'If you requested your business name through your account, your Name Request (NR) will appear in your Business Registry dashboard. If you did not request your name through your account, you will need to add your Name Request to your dashboard manually using the name your requested or your NR number. You can track the approval status of your name request on your Business Registry dashboard.',
```

Comment:
Still a typo here (the name YOU requested, not YOUR).

Reply:
Change should be in next commit, I wanted to test the general component first

Reply:
I'd rather review once, not multiple times (even if incrementally). You can always reach out to me to discuss ideas or designs before PRing it.